extensions/ql-vscode/gulpfile.ts/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  overrides: [
+    {
+      files: ["*"],
+      rules: {
+        "@typescript-eslint/no-explicit-any": "off",
+      },
+    },
+  ],
+};
```

Comment:
Oh good point, you can have multiple overrides for different sets of file paths in the same `.eslintrc.js` file. Somehow I missed that 🙈 . I'll add them there instead of in separate files.